### DISCUSSION TITLE
Private repo github

### DISCUSSION THREAD

**Discussion by omartheironman** [helmfile/helmfile#1288] [Questions]
I am using helmfile as an argocd plugin, if i'm fetching helm charts from public github repo it works, but if its private it doesn't. 

I tried execing into the sidecar container and i found out it doesn't have the access, I did see your tutorial providing PAT and gitub token, but is there a way to let helmfile use SSH key ? 

if so can you please provide me an example?

**Comment by yxxhero** [comment]
@omartheironman please post some examples.
